### PR TITLE
feat: Add AccountInfoDropdown component on design system

### PR DESCRIPTION
**Pull request by diegonzs-roll**
## What's done
# Issue
https://github.com/roll-network/roll-app/issues/109
## Tasks

- [ ] Have you written tests for new code (if applicable)?
- [ ] Have you tested the changes on all the platforms?
  - [ ] iOS
  - [ ] Android
  - [ ] Web
- [ ] Have you added stories to demonstrate the new functionality (if there is any)?

## Demo (screenshots, recordings, etc.)
https://www.loom.com/share/a97ca9a30edd4ebdaa07f85b0d54727a

### CODE REVIEW

File: packages/design-system/src/molecules/accountInfoDropdown/acountInfoDropdown.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { CircleImg } from '../../atoms'
+import { container } from '../../styles'
+import { AccountInfoDropdown, AccountInfoDropdownProps } from '.'
+
+export default {
+  title: 'Design System/Molecules/AccountInfoDropdown',
+  component: AccountInfoDropdown,
+}
+
+const Template = (args: AccountInfoDropdownProps) => (
```
Comment (teimurjan) [inline packages/design-system/src/molecules/accountInfoDropdown/acountInfoDropdown.stories.tsx:13]:
```suggestion
const Template = (props: AccountInfoDropdownProps) => (
```

File: packages/design-system/src/molecules/accountInfoDropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  name: string
+  username: string
+  children: React.ReactNode
+  options: {
+    id: string
+    iconVariant: IconVariant
+    color?: string
+    label: string
+    onPress: () => void
+  }[]
```
Comment (teimurjan) [inline packages/design-system/src/molecules/accountInfoDropdown/index.tsx:21]: Let's extract into a separate type for a better readability

```suggestion
  options: AccountInfoDropdownOption[]
```

File: packages/design-system/src/molecules/accountInfoDropdown/index.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    width: DROPDOWN_WIDTH,
+  },
+  divider: {
+    height: 1,
+  },
+})
+
+const renderListSeparator = () => (
+  <View style={[margin.mv8, container.fullWidth]} />
+)
```
Comment (teimurjan) [inline packages/design-system/src/molecules/accountInfoDropdown/index.tsx:35]: We can make it a static value so we don't create a new component each time the function is called.

```suggestion
const listSeparator = <View style={[margin.mv8, container.fullWidth]} />
```

Comment (diegonzs-roll) [reply packages/design-system/src/molecules/accountInfoDropdown/index.tsx:35]: Element is not an allow value type for ItemSeparatorComponent prop
```
Type 'Element' is not assignable to type 'ComponentType<any> | null | undefined'.

**Review by teimurjan — APPROVED** [review]

**Pull request merged**